packages/fluttium/lib/src/actions/remove_text.dart

```diff
@@ -0,0 +1,50 @@
+import 'package:flutter/services.dart';
+import 'package:fluttium/fluttium.dart';
+
+/// {@template remove_text}
+/// Remove text automatically.
+///
+/// This action can be invoked either using the short-hand version:
+///
+/// ```yaml
+/// - removeText:
+/// ```
+///
+/// Or using the verbose version:
+///
+/// ```yaml
+/// - removeText:
+///     amount: 10
+/// ```
+/// {@endtemplate}
+class RemoveText extends Action {
+  /// {@macro remove_text}
+  const RemoveText({
+    this.amount = 1,
+  });
+
+  /// The amount to remove
+  final int amount;
```

Comment:
```suggestion
  final int times;
```

nit feel free to ignore.